doc/adr/0012-rewards.md

```diff
@@ -0,0 +1,44 @@
+# Rewards
+
+Date: 2022-06-01
+
+## Status
+
+Proposed
+
+## Context
+
+Facilitators get rewarded in DAG currency for participating in consensus rounds.
+We need to figure out the algorithm for minting/sending rewards to facilitators. Algorithm
+should take dynamic network speed into account to determine the size of rewards per node.
+
+## Decision
+
+1. Each node will track the time difference between the creation of local snapshots.
+We can either store the average or keep the specific number of meaningful diffs (e.g. 100 last diffs) and calculate the average on demand.
```

Comment:
One mention here is there's also a dependency on the download process to be updated to incorporate this information -- or otherwise have nodes that are new wait to propose these values until they have the requisite information. Since there is a state dependency, either the download must pull that state from another active node -- or more simply, it could propose None's and have them filtered out until it catches up and can determine it's own local value. The second solution is probably better as it doesn't deal with issues of modifying the download or dealing with synchronization issues across the node it's downloading from.